internal/services/oracle/oracle_database_system_resource_test.go

```diff
@@ -0,0 +1,182 @@
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-09-01/dbsystems"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type DatabaseSystemResource struct{}
+
+func (a DatabaseSystemResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := dbsystems.ParseDbSystemID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.Oracle.OracleClient.DbSystems.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving %s: %+v", *id, err)
+	}
+	return pointer.To(resp.Model != nil), nil
+}
+
+func TestDatabaseSystemResource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, oracle.DatabaseSystemResource{}.ResourceType(), "test")
+	r := DatabaseSystemResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("admin_password", "initial_data_storage_size_in_gb", "pluggable_database_name", "resource_anchor_id"),
+	})
+}
+
+func TestDatabaseSystemResource_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, oracle.DatabaseSystemResource{}.ResourceType(), "test")
+	r := DatabaseSystemResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("admin_password", "initial_data_storage_size_in_gb", "pluggable_database_name", "resource_anchor_id"),
+	})
+}
+
+func TestDatabaseSystemResource_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, oracle.DatabaseSystemResource{}.ResourceType(), "test")
+	r := DatabaseSystemResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func (a DatabaseSystemResource) basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_oracle_database_system" "test" {
+  name                = "acctest%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = "%[3]s"
+  zones               = ["2"]
+  admin_password      = "testAdminPassword123##"
+  compute_count       = 4
+  compute_model       = "ECPU"
+  database_edition    = "EnterpriseEdition"
+  database_system_options {
```

Comment:
`basic` should contain `Required` properties only, `database_system_options` is `Optional`.